rest_framework/authtoken/management/commands/drf_create_token.py

```diff
@@ -16,5 +16,11 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         username = options['username']
-        token = self.create_user_token(username)
+
+        try:
+            token = self.create_user_token(username)
+        except User.DoesNotExist:
+            print('Cannot create the Token: user {0} does not exist'.format(
```

Comment:
@rpkilby yep! You are totally right and it's something I just spotted a couple of days ago and I was going to amend it. The officia documentation also suggest to use self.stdout because it's even easier to test. I just haven't had any time yet to complete it. I will try to finish during the weekend. Thanks